packages/client-search/src/types/Log.ts

```diff
@@ -68,4 +68,13 @@ export type Log = {
    * Index name of the log
    */
   readonly index: string;
+  /**
+   * Internal queries performed for this query.
+   */
+  readonly inner_queries: ReadonlyArray<{
+    readonly indexName: string;
+    readonly queryID: number;
+    readonly offset: number;
+    readonly userToken: string;
```

Comment:
are all these always present, or can they be missing @Ant-hem ?

Reply:
They could be missing.